cfgov/templates/wagtailadmin/login.html

```diff
@@ -25,9 +25,9 @@ <h1>Log in to consumerfinance.gov</h1>
             {% block login_form %}
             {% csrf_token %}
 
-            {% url 'user_welcome' as home_url %}
-            <input type="hidden" name="next" value="{{ next|default:home_url }}" />
-
+            {% if next %}
+            <input type="hidden" name="next" value="{{ next }}"/>
```

Comment:
Can we get rid of this override template entirely? If I'm right that the alternate `h1` text for build vs. other servers is the only thing we're still customizing here, it hardly seems worth it to me.

But, if we want to stick with it for now, why not revert these lines to what currently exists in this file in Wagtail 2.5.2?

```suggestion
            {% url 'wagtailadmin_home' as home_url %}
            <input type="hidden" name="next" value="{{ next|default:home_url }}" />
```

Reply:
This is a nice thought. It would be good to get rid of this override, but I don't think we can right now, for at least two reasons:

1. The default template [POSTs to the `wagtailadmin_login` view](https://github.com/wagtail/wagtail/blob/ce6b2fe97b6fada507741c6221493c6d4759afe8/wagtail/admin/templates/wagtailadmin/login.html#L29), and we override that to [instead POST to our own `cfpb_login` view](https://github.com/cfpb/cfgov-refresh/blob/1b7b9b64d70753f74487bf5f848cbed9d198cc21/cfgov/templates/wagtailadmin/login.html#L24). (It would also be good to remove this view and find other, less custom, ways to do our password/lockout validation, for example something like [django-axes](https://github.com/jazzband/django-axes).)
2. We make use of [`form.non_field_errors`](https://github.com/cfpb/cfgov-refresh/blob/1b7b9b64d70753f74487bf5f848cbed9d198cc21/cfgov/templates/wagtailadmin/login.html#L9) to show messages about [failed logins](https://github.com/cfpb/cfgov-refresh/blob/1b7b9b64d70753f74487bf5f848cbed9d198cc21/cfgov/v1/auth_forms.py#L106) and [expired passwords](https://github.com/cfpb/cfgov-refresh/blob/1b7b9b64d70753f74487bf5f848cbed9d198cc21/cfgov/v1/auth_forms.py#L121). The default template [doesn't allow such customization](https://github.com/wagtail/wagtail/blob/ce6b2fe97b6fada507741c6221493c6d4759afe8/wagtail/admin/templates/wagtailadmin/login.html#L17-L22), although it does support use of Django messages, so potentially we could use those instead.

Modifying our form to use the default next URL from Wagtail upstream seems reasonable, though. I've added a commit that does that.